projects/canopy/src/lib/card/card.component.scss

```diff
@@ -1,3 +1,5 @@
+@import '../../styles/mixins';
```

Comment:
Did you need this include? I couldn't see it referenced anywhere

Reply:
Nope, removed it.